src/pages/company-info/index.js

```diff
@@ -0,0 +1,232 @@
+import { Box, Grid } from '@mui/material'
+import React, { useContext, useEffect, useState } from 'react'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import FormShell from 'src/components/Shared/FormShell'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import useResourceParams from 'src/hooks/useResourceParams'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import toast from 'react-hot-toast'
+import { useForm } from 'src/hooks/form'
+import ImageUpload from 'src/components/Inputs/ImageUpload'
+import { getStorageData } from 'src/storage/storage'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+
+export function PostImage(postRequest, obj, initialValues) {
+  if (obj?.attachment?.file) {
+    return postRequest({
+      extension: SystemRepository.Attachment.set,
+      record: JSON.stringify(obj.attachment),
+      file: obj?.attachment?.file
+    })
+      .then(res => {
+        return res
+      })
+      .catch(e => {})
+  } else if (!obj?.attachment && initialValues?.attachment?.url) {
```

Comment:
You can know if the image was changed (dirty) from the component's onChange. Any change (edit or delete) should cause the image to resubmitted.